data/standard_features/14496-15/codecs.json

```diff
@@ -103,7 +103,7 @@
     {
       "fourcc": "avcp",
       "description": "AVC parameter set",
-      "type": "VisualSampleEntry",
+      "type": "AVCParameterSampleEntry",
```

Comment:
@DenizUgur this one fails the pipeline once we use the correct type. How should we deal with generic types? FYI `VisualSampleEntry` is defined in ISOBMFF and is a generic class which is used in derived specs with a fourCC.

Reply:
Should we add something like this to the 14496-12/codecs.json?

```json
{
      "fourcc": "",
      "description": "Generic VisualSampleEntry",
      "type": "VisualSampleEntry",
      "containers": ["stsd"],
      "syntax": "class VisualSampleEntry(codingname) extends SampleEntry (codingname){\r\n\tunsigned int(16) pre_defined = 0;\r\n\tconst unsigned int(16) reserved = 0;\r\n\tunsigned int(32)[3]\tpre_defined = 0;\r\n\tunsigned int(16)\twidth;\r\n\tunsigned int(16)\theight;\r\n\ttemplate unsigned int(32)\thorizresolution = 0x00480000;\t// 72 dpi\r\n\ttemplate unsigned int(32)\tvertresolution  = 0x00480000;\t// 72 dpi\r\n\tconst unsigned int(32)\treserved = 0;\r\n\ttemplate unsigned int(16)\tframe_count = 1;\r\n\tuint(8)[32]\tcompressorname;\r\n\ttemplate unsigned int(16)\tdepth = 0x0018;\r\n\tint(16)\tpre_defined = -1;\r\n\t// other boxes from derived specifications\r\n\tCleanApertureBox\t\t\tclap;\t\t// optional\r\n\tPixelAspectRatioBox\t\tpasp;\t\t// optional\r\n}"
    },
```

But then it will complain about missing 4CC. maybe we should add another fields to define if the type is abstract or not?

Reply:
Fixed in #68